hcn/hcnloadbalancer.go

```diff
@@ -280,13 +296,18 @@ func (loadBalancer *HostComputeLoadBalancer) RemoveEndpoint(endpoint *HostComput
 }
 
 // AddLoadBalancer for the specified endpoints
-func AddLoadBalancer(endpoints []HostComputeEndpoint, isILB bool, isDSR bool, sourceVIP string, frontendVIPs []string, protocol uint16, internalPort uint16, externalPort uint16) (*HostComputeLoadBalancer, error) {
+func AddLoadBalancer(endpoints []HostComputeEndpoint, isILB bool, isDSR bool, useMux bool, preserveDIP bool, sourceVIP string, frontendVIPs []string, protocol uint16, internalPort uint16, externalPort uint16) (*HostComputeLoadBalancer, error) {
```

Comment:
Can you modify this function params to take in LoadBalancerFlags and LoadBalancerPortMappingFlags, and remove isILB, isDSR, useMux, preserveDip etc..